src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchJsonEditorDialog.vue

```diff
@@ -99,7 +99,7 @@
                         :disabled="!newQueryName"
                         outlined
                         style="margin-right: 5px"
-                        @click="saveQuery"
+                        @click="() => saveQuery(false)"
```

Comment:
@makc-dataloop Why is this needed ? the default of this query is false so when you click saveQuery it will have a default value 
wrapping it and explicitly calling shouldnt matter ?

Reply:
@fadiDL unfortunately it receives an event for an argument and it is != false